packages/ui/src/common/utils/validation.tsx

```diff
@@ -33,7 +33,7 @@ export const maxContext = (
     }
 
     const validationValue = new BN(get(this.options.context, contextPath))
-    if (validationValue && validationValue.lt(new BN(value))) {
+    if (validationValue && validationValue.lte(new BN(value))) {
```

Comment:
The previous logic is correct:
It returns an error if `validationValue` is strictly less than `value`.
So `value` is valid when it is greater or equal to `validationValue`

So only the validation messages were wrong

Reply:
Makes sense.